.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "db-snapshot"]
 	path = db-snapshot
 	url = ../db-snapshot
+[submodule "third_party/openmrs-module-sync"]
+	path = third_party/openmrs-module-sync
+	url = git@github.com:openmrs/openmrs-module-sync.git
```

Comment:
I went for a submodule because:
- If we copy-paste the source into our directory we lose all the history
- Submodules are pinned to a specific commit - the submodule isn't updated to point at a newer commit unless you do so intentionally. So I pinned it to the version with my fix in it.

We copied xforms for different reasons, I think. It's certainly been customized a lot (See the git log for that directory).

I have to admit though, I'm not super experienced with submodules.